### DISCUSSION TITLE
[Enhancement] macOS Project on Windows!

### DISCUSSION THREAD

**Discussion by mhrastegari** [dotnet/maui#141] [General]
## Summary
This would be possible if Visual Studio and MAUI Team build some remote tools like the iOS tools which we can remote the iOS Simulatior of a macOS machine and with this we can build for all platforms in our Windows device.

**Comment by GeraudFabien** [comment]
I don't understand.
How is it different from https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/windows/connecting-to-mac/

**Comment by mhrastegari** [comment]
@GeraudFabien i'm saying we could have same remote tools of iOS for macOS projects on windows too.

**Comment by VladislavAntonyuk** [comment]
The idea is to create macOS remote simulator.

**Comment by mhrastegari** [comment]
Exactly

**Comment by davidbuckleyni** [comment]
U will always need a Mac to compile to Mac OS Apple will not change this it’s part of their security eco system.

**Comment by VladislavAntonyuk** [comment]
@davidbuckleyni the idea is to use remote Mac device and debug the app, the same we have for iOS

**Comment by dersia** [comment]
I would love to see it the other way around too, create UWP/WinUI apps from a Mac by adding tools and a Win simulator.

**Comment by McoreD** [comment]
I guess it is related - I cannot add a macOS Desktop UI project, only Console project:
https://sharex.delpach.com/2020/07/devenv_20200704_QQoKHostMp.png
This is despite Xamarin supports macOS.
